config/config_test.go

```diff
@@ -177,6 +182,128 @@ func TestInitConfig(t *testing.T) {
 	assert.Equal(t, "", param.Federation_DiscoveryUrl.GetString())
 }
 
+// Create a root config file with a Continue key pointing to a test directory
+func setupContConfig(t *testing.T, continueDir string) {
+	rootCfgDir := t.TempDir()
+
+	viper.AddConfigPath(rootCfgDir)
+	viper.SetConfigType("yaml")
+	viper.SetConfigName("pelican")
+
+	err := os.WriteFile(filepath.Join(rootCfgDir, "pelican.yaml"), []byte(fmt.Sprintf("ContinuedConfigLocation: %s\nOtherVal: bar", continueDir)), 0644)
+	require.NoError(t, err)
+	err = viper.MergeInConfig()
+	require.NoError(t, err)
+}
+
```

Comment:
Can you add a comment header here describing the test? Just want to stay consistent with everything